stdlib/2and3/cgitb.pyi

```diff
@@ -28,6 +28,6 @@ class Hook:  # undocumented
     def __call__(self, etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
     def handle(self, info: Optional[_ExcInfo] = ...) -> None: ...
 
-def handler(etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
+def handler(info: Optional[_ExcInfo]) -> None: ...
```

Comment:
Just add `= ...`, and I am happy :)

Reply:
Done, sorry about that. I should be more careful editing in my early morning